Documentation/BuildInstructionsLadybird.md

```diff
@@ -215,6 +215,22 @@ ninja -C Build/ladybird
 
 Then, based on what output you get from that, you can troubleshoot the *actual* problem you’re running into — which may involve uninstalling your current Ninja install, and then re-installing it.
 
+#### “Targets may link only to libraries. CMake is dropping the item” (when building with the Qt chrome on macOS)
```

Comment:
Hmm. Is this really a 'build instructions' kind of note?

I think in the past we have put these kind of advisories in the Troubleshooting.md file. Not that I've seen anyone reference that recently 😅

Reply:
Yeah, the Troubleshooting.md file seems like a better place for it. It’s a non-problem regardless, so not really essential info, and I think we’d not be losing much without it. But the Troubleshooting.md file is still small enough that it’s not cluttering anything up at least. And I had actually read through that file previously (so I’m existence proof that it does get read sometimes)